source/sass/buyers-guide/views/homepage.scss

```diff
@@ -147,6 +147,22 @@
     }
   }
 
+  .no-matching-products-note {
```

Comment:
let's move this into the filter.scss instead. This should not be tied to "the homepage".